x/stake/keeper.go

```diff
@@ -371,6 +371,30 @@ func (k Keeper) GetDelegatorBond(ctx sdk.Context,
 	return bond, true
 }
 
+// load all bonds
+func (k Keeper) GetBonds(ctx sdk.Context, maxRetrieve int16) (bonds []DelegatorBond) {
```

Comment:
Does this need to be exposed? I don't think we need to use this outside of specialized stake functionality

Reply:
Nope, now unexposed.